WordPress/Classes/Stores/StatsPeriodStore.swift

```diff
@@ -170,7 +183,45 @@ private extension StatsPeriodStore {
 
     func setAllAsFetching() {
         state.fetchingPostsAndPages = true
+        state.fetchingSearchTerms = true
+    }
+
+    /// This method modifies the 'Unknown search terms' row and changes its location in the array.
+    /// - Find the 'Unknown search terms' row
+    /// - Change the label
+    /// - Remove the row from the array
+    /// - Insert the row at the beginning of the array
+    /// NOTE: When the backend is updated, maybe it will return the unknown row at the top
+    /// of the array, making this unnecessary.
+    ///
+    func reorderSearchTerms(_ searchTerms: StatsGroup?) -> [StatsItem]? {
+
+        guard var searchTerms = searchTerms?.items as? [StatsItem] else {
+            return nil
+        }
+
+        // Find the row in the array
+
+        // This labelToFind matches that in WPStatsServiceRemote:operationForSearchTermsForDate
+        let labelToFind = NSLocalizedString("Unknown Search Terms", comment: "N/A. Not visible to users.")
+        let unknownSearchTerms = searchTerms.filter({ return $0.label == labelToFind })
+
+        guard let unknownSearchTermRow = unknownSearchTerms.first else {
```

Comment:
I think this "`filter`-then-`.first`-on-a-returned-collection" can be replaced by just doing `first(where:)` on `Array`: https://developer.apple.com/documentation/swift/array/1848165-first